packages/base/cards-grid.gts

```diff
@@ -384,13 +384,19 @@ class Isolated extends Component<typeof CardsGrid> {
           on: catalogEntryRef,
           eq: { ref: activeFilterRef },
         },
+        sort: [
+          {
+            by: 'createdAt',
+            direction: 'desc',
+          },
+        ],
       };
     }
     let card = await chooseCard<CatalogEntry>(
       {
         filter: {
           on: catalogEntryRef,
-          every: [{ eq: { isField: false } }],
+          every: [{ eq: { type: 'card' } }],
```

Comment:
I changed this to be clearer about what we are filtering for. But I aslo found that when isField was turned into a computed, it cudn query and I filed https://linear.app/cardstack/issue/CS-7821/filtering-by-computed-doesnt-work-at-the-first-level

Reply:
Just double checking, we don’t allow you to filter by fields that are not in the template. Is this field in the template? If not is it set to is used? I’m positive we have passing tests that exercise the ability to filter by computeds

Reply:
Confirm. Oh its another one of those. Thanks for pointing out.

I confirm within the test that that computed **WORKS** when the computed field exist within the isolated template. So, the behaviour is as you point out. Although I would add that it won't work if the computed field only exist within embedded or fitted (not sure if its something we want to consider) but not the isolated template

Reply:
So the rationale for the behavior is the "social graph problem". Imagine a boxel app that is a social graph like facebook. each person card has a linkToMany relationship to all their friends, which are also person cards. If we have no limits on what feields are included in the search doc, we will likely end up including all the people in the world in each search doc. the template is a natural way to draw boundaries on the search doc. we do have an escape valve `isUsed` which can allow a field to be in the search doc even though its not in a template (and this could get abused to create the problem that I just raised).

Reply:
How about you call this attribute `specType` instead of `type`? It is confusing because there's a real `type: "card"`. I think `isField` is helpful so we can distinguish between field defs and card defs. `specType` can then be freely used for the visual design reasons you need it for.